cloud/pkg/router/provider/rest/rest.go

```diff
@@ -201,7 +201,13 @@ func (r *Rest) GoToTarget(data map[string]interface{}, stop chan struct{}) (inte
 	if !ok || len(content) == 0 {
 		return nil, errors.New("invalid convert to []byte")
 	}
-	req, err := httpUtils.BuildRequest(http.MethodPost, r.Endpoint, bytes.NewReader(content), "", "")
+	nodeName, ok := data["nodeName"].(string)
+	if !ok {
+		err := fmt.Errorf("input data does not exist valid value \"nodeName\"")
+		klog.Warningf(err.Error())
+		nodeName = ""
```

Comment:
If not ok, the `nodeName` will not be ""? Is it needed to set the "" value here? :)

Reply:
if  return err,  the  two E2E tests  failed  in  CI.

Reply:
> if return err, the two E2E tests failed in CI.

Yes, no need to return err here. And I mean it no need to set empty value to nodename, which was originally empty.

Reply:
Ooooh, I see.   if    "nodeName, ok := data["nodeName"].(string) "  convert error  , "nodeName"  default value is empty.

I  will modify the code and push.  Thank you.